bluepysnap/circuit_validation.py

```diff
@@ -316,44 +312,44 @@ def validate_node_population(nodes_file, population_dict, name):
         list: List of errors, empty if no errors
     """
     with h5py.File(nodes_file, "r") as h5f:
-        nodes_h5 = h5f.get("nodes")
-
-        if not nodes_h5 or len(nodes_h5) == 0:
+        if "nodes" not in h5f or len(h5f["nodes"]) == 0:
```

Comment:
I'd say it's an error if the nodes file isn't actually a node file

Reply:
Yep, this is handled in schema validation.